scripts/install_moodle.sh

```diff
@@ -883,6 +884,9 @@ EOF
    #else # mssql. TODO It's missed earlier! Complete this!
    fi
 
+   # setting the local cache file directory
+   sed -i "23 a \$CFG->localcachedir = '/var/run/moodle/localcache';" /moodle/html/moodle/config.php
```

Comment:
Please do move this code to where all other `/moodle/html/moodle/config.php` modifications are done (e.g., after line 861). I really don't want this to be far apart from related code. All the code that updates `/moodle/html/moodle/config.php` must be grouped together. In fact, I'd even say they should be refactored into a function.

Also, please do consolidate all these recent 3 PRs into one. I don't think these changes warrant individual PRs. One PR for all should be enough.